### ISSUE TITLE
[BUG] Entity.SelectForm() throwing StaleElement exception

### ISSUE THREAD

**Issue by imartinflores** [Microsoft/EasyRepro#870]
**EasyRepro Version**
  - [ ] Microsoft Dynamics 365 Online Version 9.1 (9.1.x) (DB 9.1.x) online
  
**UCI or Classic Web**
  - [ ] UCI (XrmApp)

**Online or On Premise**
  - [ ] Online

**Browser**
  - [ ] Chrome

**Describe the bug**  
Using Entity.SelectForm() is throwing StaleElement exception.

**Code to reproduce**  
 XrmApp.Entity.SelectForm(formName);

```csharp
 XrmApp.Entity.SelectForm(formName);
```
 -->
 
**Expected behavior**  
Select the correct form and wait until page is loaded

**Screenshots**  

**Additional context**  

Stack Trace: 
    OpenQA.Selenium.StaleElementReferenceException: stale element reference: element is not attached to the page document
  (Session info: chrome=81.0.4044.138)
    RemoteWebDriver.UnpackAndThrowOnError(Response errorResponse)
    RemoteWebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
    RemoteWebElement.Execute(String commandToExecute, Dictionary`2 parameters)
    RemoteWebElement.get_Location()
    InteractiveBrowser.EventDriver_ElementClicked(Object sender, WebElementEventArgs e)
    EventFiringWebDriver.OnElementClicked(WebElementEventArgs e)
    EventFiringWebElement.Click()
    <>c.<ClickWhenAvailable>b__2_0(IWebElement e)
    SeleniumExtensions.WaitUntil(ISearchContext driver, Func`2 searchFunc, Nullable`1 timeout, Action`1 successCallback, Action failureCallback)
    SeleniumExtensions.WaitUntilClickable(ISearchContext driver, By by, Nullable`1 timeout, Action`1 successCallback, Action failureCallback)
    SeleniumExtensions.WaitUntilClickable(ISearchContext driver, By by, Nullable`1 timeout, Action`1 successCallback, String exceptionMessage)
    SeleniumExtensions.ClickWhenAvailable(ISearchContext driver, By by, Nullable`1 timeout, String errorMessage)
    <>c__DisplayClass158_0.<SelectForm>b__0(IWebDriver driver)
    DelegateBrowserCommand`1.ExecuteCommand(IWebDriver driver, Object[] params)
    BrowserCommand`1.Execute[T1,T2,T3,T4,T5,T6,T7,T8,T9](IWebDriver driver, T1 p1, T2 p2, T3 p3, T4 p4, T5 p5, T6 p6, T7 p7, T8 p8, T9 p9)
    BrowserPage.Execute[TResult](BrowserCommandOptions options, Func`2 delegate)
    WebClient.SelectForm(String formName)
    Entity.SelectForm(String formName)

**Comment by TYLEROL** [comment]
@imartinflores - I made changes to the form selector element references in #833 which is currently merged into the develop branch.

Can you confirm whether this issue reproduces in develop? I am not able to reproduce with the latest develop changes.

-Tyler

**Comment by imartinflores** [comment]
@TYLEROL  Thanks for the update, I will have a look and confirm. We are using the latest version in the Nuget, let me try again and will let you know. The changes you made are not in Nuget yet, right ?

Thanks.

**Comment by TYLEROL** [comment]
@imartinflores - That's correct. This change is only in develop, though I plan to release an updated NuGet package either today or tomorrow, depending on when #864 is merged to develop.

-Tyler

**Comment by imartinflores** [comment]
Perfect, thanks a lot @TYLEROL

**Comment by TYLEROL** [comment]
This has been merged into releases/v9.1 and the latest 9.1.0.17138 NuGet package.
